howto-build-ios.md

```diff
@@ -84,9 +88,22 @@ Your CakeWallet binary will be built with cryptographic salts, which are used fo
 
 `$ flutter packages pub run tool/generate_new_secrets.dart`
 
-Then we need to generate localization files and mobx models.
+Your CakeWallet binary will be built with cryptographic salts, which are used for secure encryption of your data. You may generate these secret salts with the following command:
+
+`$ flutter packages pub run tool/generate_new_secrets.dart`
+
+If the command above fails, add `--force` flag and run it again.
+
+*Note*: You should skip this step if you've already done this once. Else, you
+will not be able to log into your existing wallet with the new secrets.
+
+Then we need to generate localization files. If this command fails, add `--force` flag and run it again.
+
+`$ flutter packages pub run tool/generate_localization.dart`
```

Comment:
It looks like this note is for the localization script. But I believe it is meant for the secrets generation script?

Generally speaking anyway, this how-to is for first time use, it appears. Only the secrets generation part will cause problems if re-run but most of the scripts here only need to be run once, really.

Reply:
It's good to mention potential gotchas so anyone running this for the first time wouldn't try to pull their hair.